packages/blade/src/components/VisuallyHidden/VisuallyHidden.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import {
+  Title as StorybookTitle,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs';
+import { Highlight } from '@storybook/design-system';
+import type { ReactElement } from 'react';
+import { EyeIcon } from '../Icons';
+import BaseText from '../Typography/BaseText';
+import type { VisuallyHiddenProps } from './VisuallyHidden';
+import VisuallyHiddenComponent from './';
+
+const Page = (): ReactElement => {
+  return (
+    <>
+      <StorybookTitle />
+      <Subtitle>
+        The VisuallyHidden component makes content hidden from sighted users but available for
+        screen reader users
+      </Subtitle>
+      <br />
+      <br />
+      <StorybookTitle>Usage</StorybookTitle>
+      <Highlight language="tsx">{`import { VisuallyHidden } from '@razorpay/blade/components' \nimport type { VisuallyHiddenProps } from '@razorpay/blade/components'`}</Highlight>
+      <StorybookTitle>Example</StorybookTitle>
+      <Primary />
+      <StorybookTitle>Properties</StorybookTitle>
+      <ArgsTable story={PRIMARY_STORY} />
+      <Stories />
+    </>
+  );
+};
+
+const VisuallyHiddenStoryMeta: Meta<VisuallyHiddenProps> = {
+  title: 'Components/Accessibility/VisuallyHidden',
+  component: VisuallyHiddenComponent,
+  args: { children: 'Visually hidden content' },
+  parameters: {
+    docs: {
+      page: () => <Page />,
+    },
+  },
+};
+
+const VisuallyHiddenTemplate: ComponentStory<typeof VisuallyHiddenComponent> = () => {
+  return (
+    <button>
+      <EyeIcon color="action.icon.link.active" size="large" />
```

Comment:
Should we use the eye icon as an example? It would probably be better to give a real life example where we could have a button or a nav element with the text "Skip to content" which when entered on leads to a content.
For e.g. check the [bbc website](https://www.bbc.com/) or even JIRA

Reply:
I guess the example isn't quite optimal, let me think of something better.  🤔 

> be better to give a real life example where we could have a button or a nav element with the text "Skip to content" 
We can do that 🤔 but we will also be having a separate <SkipNav /> component don't you think the storybook examples will get a bit redundant? 

Button with an hidden icon + visually hidden text is also a valid usecase tho.

Reply:
I tried out this example, CustomCheckbox with hidden input 😅 feels too complex tho, Let's just have Icon button only. I think it conveys the usage. 

```ts
const VisuallyHiddenTemplate: ComponentStory<typeof VisuallyHiddenComponent> = () => {
  const [checked, setChecked] = React.useState(false);
  return (
    <span>
      <VisuallyHiddenComponent>
        <input
          checked={checked}
          onChange={(): void => setChecked(!checked)}
          type="checkbox"
          id="c1"
        />
      </VisuallyHiddenComponent>
      <span style={{ outline: '1px solid red' }}>
        {checked ? '✅ ' : '❎ '}
        <label htmlFor="c1">Custom checkbox with hidden input</label>
      </span>
    </span>
  );
};
```

Reply:
Done.